addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -340,31 +340,32 @@ def _compute_date_closed(self):
             if not applicant.stage_id.hired_stage:
                 applicant.date_closed = False
 
-    @api.model
-    def create(self, vals):
-        if vals.get('department_id') and not self._context.get('default_department_id'):
-            self = self.with_context(default_department_id=vals.get('department_id'))
-        if vals.get('user_id'):
-            vals['date_open'] = fields.Datetime.now()
-        if vals.get('email_from'):
-            vals['email_from'] = vals['email_from'].strip()
-        res = super().create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get('department_id') and not self._context.get('default_department_id'):
+                self = self.with_context(default_department_id=vals.get('department_id'))
```

Comment:
This will override the `default_department_id` context key for all records created with the same value :thinking: